Makefile

```diff
@@ -797,8 +797,11 @@ test-postgres: test-postgres-docker
 test-migrations: test-postgres-docker
 	echo "--- test migrations"
 	set -euo pipefail
-	COMMIT_FROM=$(shell git rev-parse --short HEAD)
-	COMMIT_TO=$(shell git rev-parse --short main)
+	COMMIT_FROM=$(shell git log -1 --format='%h' HEAD)
+	echo "COMMIT_FROM=$${COMMIT_FROM}"
```

Comment:
Pro-tip: `declare` is pretty nice (`declare -p COMMIT_FROM COMMIT_TO`. Not necessarily for here, but future reference.